CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Yourloops data visualization for diabetes device data
 This node package is a dependency of Blip and was forked from tidepool/sundial.
 
+## Unreleased
```

Comment:
Don't you want to release this version so it can be used in other projects?

Reply:
I'm not sure to understand what you mean